src/main/java/org/broadinstitute/hellbender/engine/ReadShard.java

```diff
@@ -147,9 +160,17 @@ public boolean containsStartPosition( final Locatable loc ) {
      */
     @Override
```

Comment:
Done -- copied the clarification on order of operations to all affected methods.